### PR TITLE
Spotty change branch

### PR DESCRIPTION
**Pull request by skytribe**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/skydiving/skydiving.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+And I went to an airport  and saw someone jumping out of a plane and open and parachute and though i must try that.
```
Comment (casey-collab) [inline english/skydiving/skydiving.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/skydiving/skydiving.md (hunk 1 above)
Comment (skytribe) [inline english/skydiving/skydiving.md]: Added a link in marshmallow.md which will now link to this chapter.

**Pull request closed without merging**